wildcard/media/convert.py

```diff
@@ -126,6 +126,7 @@ def info(self, filepath):
         cmd = [self.binary, filepath]
         result = {}
         for line in self._run_command(cmd, or_error=True).splitlines():
+            line = line.decode()
```

Comment:
I think the plan is to keep compatibility with Python2.
For this reason I would decode the output only if we are not on Python 2.
Otherwise on Python 2 you might have issues in the lines below because there will be a mixture of text and bytes.
The other option is to use text also in the following lines.

Reply:
I have added a check via six.PY3